internal/telemetry/sensitivemetadataallowlist/sensitivemetadataallowlist.go

```diff
@@ -90,9 +174,22 @@ func (e EventTypes) Redact(event *telemetrygatewayv1.Event) redactMode {
 // IsAllowed indicates an event is on the sensitive telemetry allowlist, and the fields that
 // are allowed.
 func (e EventTypes) IsAllowed(event *telemetrygatewayv1.Event) ([]string, bool) {
-	key := fmt.Sprintf("%s.%s", event.GetFeature(), event.GetAction())
-	allowedKeys, allowed := e.index[key]
-	return allowedKeys, allowed
+	var key string
+	if event.GetAction() == "%" {
+		key = event.GetFeature()
+		allowedKeys, allowed := e.index[key]
+		if allowed {
+			return allowedKeys, allowed
+		}
+	} else {
+		key = fmt.Sprintf("%s.%s", event.GetFeature(), event.GetAction())
```

Comment:
match https://github.com/sourcegraph/sourcegraph/pull/63325/files#r1649431906

```suggestion
		key = fmt.Sprintf("%s/%s", event.GetFeature(), event.GetAction())
```

Reply:
updated!